docs/tutorials/01_introductory/viz_earth_animation.py

```diff
@@ -0,0 +1,163 @@
+utils.rotate(earth_actor, (-90, 1, 0, 0))
+
+##############################################################################
+# The ShowManager class is the interface between the scene, the window and the
+# interactor.
+
+showm = window.ShowManager(scene,
+                           size=(900, 768), reset_camera=False,
+                           order_transparent=True)
+
+##############################################################################
+# Next, let's focus on creating the animation.
+# We can determine the duration of animation with using the ``counter``.
+# Use itertools to avoid global variables.
+
+counter = itertools.count()
+
+##############################################################################
+# The timer will call this user defined callback every 200 milliseconds. The
+# application will exit after the callback has been called 200 times.
+
+scene.set_camera(position=(0.24, 0.00, 4.34), focal_point=(0.00, 0.00, 0.00),
+                 view_up=(0.00, 1.00, 0.00))
+
+##############################################################################
+# Let's create a sphere actor to add to the Earth. We will place this sphere
+# on the Earth's surface on Bloomington, home of FURY's headquarters!
+
+center = [[-0.39, 0.3175, 0.025]]
+radius = 0.002
+sphere_actor = actor.sphere(center, window.colors.blue_medium, radius)
+
+##############################################################################
+# Also creating a text actor to add below the sphere.
+
+text_actor = actor.text_3d("Bloomington, Indiana", (-0.42, 0.31, 0.03),
+                           window.colors.white, 0.004)
+utils.rotate(text_actor, (-90, 0, 1, 0))
+
+##############################################################################
+# Let's also import a model of a satellite to visualize circling the moon.
+
+fetch_viz_models()
+filename = read_viz_models("satellite_obj.obj")
+satellite = io.load_polydata(filename)
+satellite_actor = utils.get_actor_from_polydata(satellite)
+
+satellite_actor.SetPosition(-0.75, 0.1, 0.4)
+satellite_actor.SetScale(0.005, 0.005, 0.005)
+
+
+##############################################################################
+# In the ``timer_callback`` function, use if statements to specify when
+# certain events will happen in the animation, based on the position that
+# the counter is at. So, for example, the earth actor will continue to
+# rotate while the count is less than 450.
+
+def timer_callback(_obj, _event):
+    cnt = next(counter)
+    showm.render()
+    if cnt < 450:
+        utils.rotate(earth_actor, (1, 0, 1, 0))
+    if cnt % 5 == 0 and cnt < 450:
+        showm.scene.azimuth(-1)
+    if cnt == 300:
+        scene.set_camera(position=(-3.679, 0.00, 2.314),
+                         focal_point=(0.0, 0.35, 0.00),
+                         view_up=(0.00, 1.00, 0.00))
+    if cnt > 300 and cnt < 450:
+        scene.zoom(1.01)
+    if cnt == 450:
+        print(scene.get_camera())
```

Comment:
Can you remove the `print` and the `if`? I suppose it was for debugging purposes. Thanks!